js_modules/dagster-ui/packages/ui-core/src/runs/useRunsForTimeline.tsx

```diff
@@ -60,9 +62,9 @@ export const useRunsForTimeline = ({
     () =>
       new HourlyDataCache<RunTimelineFragment>(
         localCacheIdPrefix ? `${localCacheIdPrefix}-useRunsForTimeline` : false,
+        JSON.stringify(filter),
```

Comment:
Hmm, this is weird to me, I think the filtered version should live in a separate cache because I don't want it to evict the main Run timeline from the cache

Reply:
Hmm I like that, I can put ALL of the filtered ones into the same second cache, just didn't want to make a separate cache for each separate filters since nothing ever deletes the indexdbs.

Reply:
Yeah good call, I broke this apart so that if `filter` is present it uses a `useRunsForTimeline-filtered` key instead of interfering with the main one. 

Also verified that the limit=3 is working on the new one and visiting a 4th backfill evicts the oldest item from the LRU cache 🙌 

<img width="1728" alt="image" src="https://github.com/dagster-io/dagster/assets/1037212/ae495efc-189c-4f20-8a91-0b6436f0e147">


<img width="1728" alt="image" src="https://github.com/dagster-io/dagster/assets/1037212/520cfda9-dac1-4100-bd50-c94d73a5d70a">

Reply:
I don't see that in the logic above.